lib/event.rb

```diff
@@ -1,18 +1,14 @@
 # Base class for events
 class Event
   COMMUNITY_CHANNEL = 'G3MD48QTD'
+  MENTORS_INTERNAL_CHANNEL = 'G04CRMCT4'
 
   def initialize(data, token: nil, logger: nil)
     @data = data
     @token = token
     @logger = logger
   end
 
-  # Make sure we are in dev mode unless explicitly disabled
-  def production_mode?
```

Comment:
Technically, since both `team_join.rb` and `mentor_joined_channel.rb` are inheriting from `Event`, this method could remain in `event.rb`.

It works having it in `common.rb`, too.  If moving forward we feel the lion's share of the common code will live in `event.rb`, and classes are going to continue to inherit from `event.rb`, perhaps we want that method to remain there?

Your call, though.